### DISCUSSION TITLE
Add to multiple selection

### DISCUSSION THREAD

**Discussion by Mariete124** [vzhd1701/csv2notion#2] [Q&A]
Hey I've been trying the tool and it is awesome. 

Just one more question: ¿Is there anyway to add to a mulitple selection one more?
![image](https://user-images.githubusercontent.com/94454662/171827829-d16566fa-6aae-4931-889e-807b2fb747ae.png)
I have this and I want the updater to just add one more to be like this
![image](https://user-images.githubusercontent.com/94454662/171827936-c07d53c0-8dae-49c0-b9cb-331d1aa8855d.png)
But instead it just replace the hole multiple selecction
![image](https://user-images.githubusercontent.com/94454662/171828018-f339248f-4de3-4817-8d53-8a7ac0356cfa.png)
Thanks for everything!!

**Comment by vzhd1701** [comment]
You must put "Breakdown, EDL" into CSV column if you want both of those values in the database column.

**Reply by Mariete124** [reply]
And is there a way to just add it? So if I have one that didn't had the breakdown put on it the updater doesn't add it, just the EDL

**Reply by vzhd1701** [reply]
No, not with the current logic. The tool is supposed to sync data from CSV into Notion DB.

**Reply by Mariete124** [reply]
Okay, thanks anyway seriously great job!!!
